lib/puppet/type/grub_menuentry.rb

```diff
@@ -8,6 +8,9 @@
 # Based on work by Dominic Cleal
 
 Puppet::Type.newtype(:grub_menuentry) do
+  require 'puppet/parameter/boolean'
+  require 'puppet/property/boolean'
```

Comment:
IIRC Puppet::Property::Boolean doesn't work. If you try to set a *property* to the boolean `false`, puppet won't treat it any differently from if you had set it to `undef` or hadn't set it at all.

Reply:
See https://github.com/theforeman/puppet-pulp/commit/7fdb52396ae6200d2dcfefc248e327c5548c25c0

Reply:
Well neat :-|. I'll work on getting that updated.